examples/src/main/java/com/mapbox/navigation/examples/PermissionsHelper.kt

```diff
@@ -0,0 +1,118 @@
+package com.mapbox.navigation.examples
+
+import android.Manifest
+import android.app.Activity
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import com.mapbox.android.core.permissions.PermissionsListener
+import java.util.ArrayList
+
+private const val COARSE_LOCATION_PERMISSION = Manifest.permission.ACCESS_COARSE_LOCATION
+private const val FINE_LOCATION_PERMISSION = Manifest.permission.ACCESS_FINE_LOCATION
+private const val BACKGROUND_LOCATION_PERMISSION = "android.permission.ACCESS_BACKGROUND_LOCATION"
+const val LOCATION_PERMISSIONS_REQUEST_CODE = 1612
+
+/**
+ * It's a temporary replacement for PermissionsManager from com.mapbox.android.core.permissions
+ * See https://github.com/mapbox/mapbox-events-android/issues/490
+ */
+class PermissionsHelper(private val listener: PermissionsListener?) {
```

Comment:
```suggestion
class LocationPermissionsHelper(private val listener: PermissionsListener?) {
```
do we use it only for location permissions?

Reply:
+